### PR TITLE
Remove unused columns

### PR DESCRIPTION
**Pull request by chakrabortyr**
## Description
Removed start event ids from the cloud transient table. Also made the wallduration datatype consistent between transient and aggregate tables.

## Motivation and Context
Event ids are no longer part of any unique constraints.

## Tests performed
Made sure the existing numbers weren't changed.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project as found in the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: configuration/etl/etl_tables.d/cloud_common/cloudfact_by_.json [hunk 1]
```diff
@@ -102,7 +102,7 @@
                 "comment": "FACT: Number of sessions ended over a given period."
             },{
                 "name": "wallduration",
-                "type": "decimal(18,0)",
+                "type": "bigint(18)",
```
Comment (jpwhite4) [inline configuration/etl/etl_tables.d/cloud_common/cloudfact_by_.json]: is 18 digits big enough?

Comment (smgallo) [reply configuration/etl/etl_tables.d/cloud_common/cloudfact_by_.json]: Why the change from decimal to bigint?

Comment (plessbd) [reply configuration/etl/etl_tables.d/cloud_common/cloudfact_by_.json]: What is the point of `(18)`?

Seems arbitrary in the case of an int.

https://dev.mysql.com/doc/refman/5.5/en/numeric-type-overview.html
```
M indicates the maximum display width for integer types. The maximum display width is 255. Display width is unrelated to the range of values a type can contain, as described in Section 11.2, “Numeric Types”. For floating-point and fixed-point types, M is the total number of digits that can be stored. 
```

Comment (chakrabortyr) [reply configuration/etl/etl_tables.d/cloud_common/cloudfact_by_.json]: Yes, as per our conversation it should scale to a sufficiently high order of magnitude.

Comment (plessbd) [reply configuration/etl/etl_tables.d/cloud_common/cloudfact_by_.json]: 18 digits in this case is just a display and seems arbitrary, I would just remove it and use the default (which for an unsigned int is 20 anyway)

If your going to make it consistent make it make sense, I would say update the others to match as well.

File: configuration/etl/etl_tables.d/cloud_common/cloudfact_by_.json [hunk 2]
```diff
@@ -62,17 +62,17 @@
...
             },{
                 "name": "core_time",
-                "type": "bigint(42)",
+                "type": "bigint(20) unsigned",
                 "nullable": false,
                 "comment": "FACT: Core hours reserved by a VM."
             },{
                 "name": "memory_reserved",
-                "type": "bigint(42)",
+                "type": "bigint(20) unsigned",
```
Comment (plessbd) [inline configuration/etl/etl_tables.d/cloud_common/cloudfact_by_.json]: please update the comment as to what the value stored in here is (bytes, KB, MB, GB)
Same with storage below

Comment (plessbd) [reply configuration/etl/etl_tables.d/cloud_common/cloudfact_by_.json]: Reason for that is, it doesnt make sense to store memory in bytes, and if it is stored in GB it doesnt make sense to store this as a bigint when we wouldnt be able to ever have that large of a number.

Same with storage...

File: configuration/etl/etl_tables.d/cloud_common/cloud_transient.json [hunk 3]
```diff
@@ -71,12 +66,12 @@
             },
             {
                 "name": "start_time_ts",
-                "type": "bigint(18)",
+                "type": "bigint(20) unsigned",
```
Comment (plessbd) [inline configuration/etl/etl_tables.d/cloud_common/cloud_transient.json]: technically the start_time_ts could be negative, but I dont think we have any timestamps from pre 1970...

**Comment by plessbd** [comment]
If you are going to make changes other than remving columns the title of this should probably be written as "database table cleanup/updates"

File: configuration/etl/etl_tables.d/cloud_common/cloudfact_by_.json (hunk 2 above)
Comment (jpwhite4) [reply configuration/etl/etl_tables.d/cloud_common/cloudfact_by_.json]: What about systems that have different word sizes? E.g. the UNIVAC III has 25 bit word size.

**Comment by jpwhite4** [comment]
Also won't someone please think of the children!

File: configuration/etl/etl_tables.d/cloud_common/cloudfact_by_.json (hunk 1 above)
Comment (plessbd) [reply configuration/etl/etl_tables.d/cloud_common/cloudfact_by_.json]: Just to note
`bigint(20)` is how it is stored if you dont specify the display width, so `bigint(20)` will need to be what is stored in teh JSON file to allow for the detection of changes in etl v2

**Pull request merged**